src/main/java/org/folio/rest/impl/StorageHelper.java

```diff
@@ -61,4 +71,16 @@ protected static PostgresClient postgresClient(Context vertxContext, Map<String,
     return PostgresClient.getInstance(vertxContext.owner(), TenantTool.tenantId(okapiHeaders));
   }
 
+  protected static <T> void postSync(String table, List<T> entities, Map<String, String> okapiHeaders,
```

Comment:
Is this method intended to eventually become part of PgUtil?

Reply:
Yes.